cli/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "multibase-cli"
+version = "0.1.0"
+authors = ["David Craven <david@craven.ch>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+async-std = "0.99.4"
```

Comment:
10/10 for using async-std :)

Reply:
but on the other hand I am wondering, do we actually need it, given that there is very little that the cli actually does